Userland/Libraries/LibJS/MarkupGenerator.cpp

```diff
@@ -161,13 +164,13 @@ void MarkupGenerator::error_to_html(Object const& object, StringBuilder& html_ou
     auto message = object.get_without_side_effects(vm.names.message).value_or(js_undefined());
 
     if (name.is_accessor() || message.is_accessor()) {
-        auto const& stringified_object = Value(&object).to_string_without_side_effects();
+        auto stringified_object = Value(&object).to_string_without_side_effects();
         html_output.append(wrap_string_in_style(String::formatted("Uncaught {}", stringified_object), StyleType::Invalid));
     } else {
         auto name_string = name.to_string_without_side_effects();
         auto message_string = message.to_string_without_side_effects();
         html_output.append(wrap_string_in_style(String::formatted("Uncaught [{}]: ", name_string), StyleType::Invalid));
-        html_output.appendff("{}<br>"sv, message_string.is_empty() ? "\"\"" : escape_html_entities(message_string));
+        html_output.appendff("{}<br>", escape_html_entities(message_string));
```

Comment:
Kind of the same question here, why is the empty check removed? Can message string not be empty?

Reply:
Yep, message string can be empty. Don't know why I decided to remove it. I'll return it back
<img width="392" alt="Screen Shot 2022-10-09 at 22 49 25" src="https://user-images.githubusercontent.com/26363017/194774338-8a8e4046-a284-41b1-99c5-f393a94855d3.png">